### ISSUE TITLE
Camera

### ISSUE THREAD

**Issue by HidenoriMatsubayashi** [sony/flutter-elinux-plugins#6]
Create camera plugin

**Comment by Nagendra-Bankupalli** [comment]
@HidenoriMatsubayashi , is camera plugin depends on video player ?

**Comment by HidenoriMatsubayashi** [comment]
No, it doesn't, but using GStreamer is the same with the video_player plugin.

**Comment by Nagendra-Bankupalli** [comment]
@HidenoriMatsubayashi , Thanks, I understand its gst-camerabin plugin. is camera.dart still in progress? as I could not fine this file ?

**Comment by HidenoriMatsubayashi** [comment]
You are trying to cross-build, but does self-building (on x64 for x64) sucess?

**Comment by Nagendra-Bankupalli** [comment]
NO, it gave same error, 

$ cd path/packages/camera/example
$ flutter-elinux pub get => success 
$ flutter-elinux build elinux --debug

💪 Building with sound null safety 💪

elinux/flutter/generated_plugin_registrant.dart:11:22: Error: Method not found: 'PathProviderELinux.register'.
  PathProviderELinux.register();
                     ^^^^^^^^
Exception
Building an eLinux application with wayland backend in debug mode for x64 target...         8.5s
The build failed.

**Comment by HidenoriMatsubayashi** [comment]
Are you using the latest version of flutter-elinux? If not, could you use the latest version?

**Comment by Nagendra-Bankupalli** [comment]
Thanks , it works with latest version

**Comment by HidenoriMatsubayashi** [comment]
> adding "upgrade" switch would be helpful

Than you for your feedback. I'll consider to add it!

**Comment by Nagendra-Bankupalli** [comment]
The default orientation of the camera is Portrait, I could change to landscape by forcing it to landscape, as most of embedded platforms are not equipped with sensors, keeping default orientation to landscape would be better 
 I guess. Please let me know if my understanding is wrong

**Comment by HidenoriMatsubayashi** [comment]
Thank you for your feedback.

> as most of embedded platforms are not equipped with sensors, keeping default orientation to landscape would be better

Yes, I agree with you. Are you using the example of camera plugin or your own camera app?

**Comment by Nagendra-Bankupalli** [comment]
I am using plugin example.

**Comment by HidenoriMatsubayashi** [comment]
@Nagendra-Bankupalli 

Thanks a lot. This is a bug, so I fixed it https://github.com/sony/flutter-elinux-plugins/pull/36

**Comment by Arna-Maity** [comment]
@HidenoriMatsubayashi Does the camera plugin support multiple cameras?

**Comment by HidenoriMatsubayashi** [comment]
No, not yet. It's one of TODOs.
